packages/docusaurus-theme-translations/locales/tk/theme-common.json

```diff
@@ -0,0 +1,74 @@
+  "theme.NotFound.p1": "Gynansakda, ýüzlenen sahypaňyz tapylmady",
+  "theme.NotFound.p2": "Web sahypanyň dolandyryjylaryna habar bermegiňizi sizden haýyş edýäris",
+  "theme.NotFound.title": "Sahypa tapylmady",
+  "theme.TOCCollapsible.toggleButtonLabel": "Bu sahypanyň mazmuny",
+  "theme.admonition.caution": "duýduryş",
+  "theme.admonition.danger": "howp",
+  "theme.admonition.info": "maglumat",
+  "theme.admonition.note": "bellik",
+  "theme.admonition.tip": "maslahat",
+  "theme.admonition.warning": "duýduryş",
+  "theme.blog.archive.description": "Arhiw",
+  "theme.blog.archive.title": "Arhiw",
+  "theme.blog.paginator.navAriaLabel": "Blog sahypasynda nawigasiýa",
+  "theme.blog.paginator.newerEntries": "Täze ýazgylar",
+  "theme.blog.paginator.olderEntries": "Köne ýazgylar",
+  "theme.blog.post.paginator.navAriaLabel": "Blog ýazgysynyň sahypasyndaky nawigasiýa",
+  "theme.blog.post.paginator.newerPost": "Täze ýazgy",
+  "theme.blog.post.paginator.olderPost": "Köne ýazgy",
+  "theme.blog.post.plurals": "{count} ýazgy|{count} ýazgylar|{count} ýazgylary",
+  "theme.blog.post.readMore": "Dowamyny okamak",
+  "theme.blog.post.readMoreLabel": "{title} hakda doly okamak",
+  "theme.blog.post.readingTime.plurals": "{readingTime} min okamak|{readingTime} min okamak|{readingTime} min okamak",
+  "theme.blog.sidebar.navAriaLabel": "Soňky blog barada nawigasiýa",
+  "theme.blog.tagTitle": "\"{tagName}\" bilen {nPosts} ýazgylar",
+  "theme.colorToggle.ariaLabel": "Gijeki ýa-da gündizki temany saýlamak (häzirki wagtda {mode} ulanylýar)",
+  "theme.colorToggle.ariaLabel.mode.dark": "Gijeki tema",
+  "theme.colorToggle.ariaLabel.mode.light": "Gündizki tema",
+  "theme.common.editThisPage": "Bu sahypany üýtgetmek",
+  "theme.common.headingLinkTitle": "{heading} sahypa göni geçiň",
+  "theme.common.skipToMainContent": "Esasy mazmuna geç",
+  "theme.docs.DocCard.categoryDescription.plurals": "{count} element|{count} elementler|{count} elementler",
+  "theme.docs.breadcrumbs.home": "Baş sahypa",
+  "theme.docs.breadcrumbs.navAriaLabel": "Häzirki sahypa üçin nawigasiýa",
+  "theme.docs.paginator.navAriaLabel": "Dokument sahypasy",
+  "theme.docs.paginator.next": "Indiki sahypa",
+  "theme.docs.paginator.previous": "Öňki sahypa",
+  "theme.docs.sidebar.closeSidebarButtonAriaLabel": "Nawigasiýa panelini ýap",
+  "theme.docs.sidebar.collapseButtonAriaLabel": "Gapdal paneli ýygnamak",
+  "theme.docs.sidebar.collapseButtonTitle": "Gapdal paneli ýygnamak",
+  "theme.docs.sidebar.expandButtonAriaLabel": "Gapdal paneli giňeltmek",
+  "theme.docs.sidebar.expandButtonTitle": "Gapdal paneli giňeltmek",
+  "theme.docs.sidebar.navAriaLabel": "Dokumentleriň gapdal paneli",
+  "theme.docs.sidebar.toggleSidebarButtonAriaLabel": "Navigasiýa panelini üýtgetmek",
+  "theme.docs.tagDocListPageTitle": "\"{tagName}\" bilen {nDocsTagged} dokumentler",
+  "theme.docs.tagDocListPageTitle.nDocsTagged": "1-nji sahypa|{count}-nji(y) sahypa|{count}-nji(y) sahypa",
```

Comment:
2 forms instead of 3?

Reply:
here plural form doesn't change itself, but as grammatically plural changes according to cases I thought it would be correct to put the 3rd form regardless of its change